homeassistant/components/light/device_action.py

```diff
@@ -100,6 +117,24 @@
                 )
             )
 
+        if supported_features & SUPPORT_FLASH:
+            actions.extend(
+                (
+                    {
+                        CONF_TYPE: TYPE_FLASH_SHORT,
+                        "device_id": device_id,
+                        "entity_id": entry.entity_id,
+                        "domain": DOMAIN,
+                    },
+                    {
+                        CONF_TYPE: TYPE_FLASH_LONG,
+                        "device_id": device_id,
+                        "entity_id": entry.entity_id,
+                        "domain": DOMAIN,
+                    },
+                )
+            )
```

Comment:
Let's make this 1 action with short/long as an option.

Reply:
Done, default is flash short